### PR TITLE
Property store changes

### PR DESCRIPTION
**Pull request by namrata1012**

### CODE REVIEW

File: www/src/pages/table/NodeSearchComponent.tsx [hunk 1]
```diff
@@ -4,10 +4,12 @@ import { Select, Spin } from "antd";
 import { observer } from "mobx-react";
 import React, { useMemo, useState } from "react";
 import { SequentialFuncInvoker } from "../../misc";
-import { ClassTextSearchResult } from "../../models/ontology/ClassStore";
```
Comment (punith300i) [inline www/src/pages/table/NodeSearchComponent.tsx:7]: rename it to ClassTextSearchResult, as we are using same naming convention for Entity and properties.

File: www/src/pages/table/NodeSearchComponent.tsx [hunk 2]
```diff
@@ -128,19 +128,13 @@ export const NodeSearchComponent = withStyles(styles)(
         classStore
           .fetchSearchResults(query)
           .then((data) => {
-            data.forEach((searchResult: ClassTextSearchResult) => {
```
Comment (punith300i) [inline www/src/pages/table/NodeSearchComponent.tsx:131]: rename it to ClassTextSearchResult, as we are using same naming convention for Entity and properties.

File: www/src/pages/table/NodeSearchComponent.tsx [hunk 3]
```diff
@@ -71,9 +73,7 @@ export const NodeSearchComponent = withStyles(styles)(
         type: "class",
         id: "",
         label: (
-          <div>
```
Comment (punith300i) [inline www/src/pages/table/NodeSearchComponent.tsx:74]: remove the div tags and just use the Spin component available from ant-d. hence, the custom SpinComponent can be removed.

File: www/src/pages/table/NodeSearchComponent.tsx [hunk 4]
```diff
@@ -171,7 +165,7 @@ export const NodeSearchComponent = withStyles(styles)(
             console.log(option);
             if (option.type == "class") {
               classStore.fetchById(option.id).then(() => {
-                onSelect({ type: option.type, id: option.id });
+                onSelect({ type: "class", id: option.id });
```
Comment (punith300i) [inline www/src/pages/table/NodeSearchComponent.tsx:168]: we need to keep this option.type as we have existing nodes in the kg.

File: www/src/pages/table/NodeSearchComponent.tsx [hunk 5]
```diff
@@ -128,19 +128,13 @@ export const NodeSearchComponent = withStyles(styles)(
...
                 label: (
-                  <div>
-                    <p style={{ color: "blue" }}>
-                      {searchResult.label} ({searchResult.id})
-                    </p>
-                    <p style={{ fontSize: 12, marginTop: -5 }}>
-                      {searchResult.description}
-                    </p>
-                  </div>
+                  <LabelComponent id={searchResult.id} label={searchResult.label}
```
Comment (punith300i) [inline www/src/pages/table/NodeSearchComponent.tsx:136]: Since we don't display uri in the label, we can remove it in the Label Component

**Review by punith300i — COMMENTED** [review]
Code review review completed. Please address the following comments.

File: www/src/pages/table/NodeSearchComponent.tsx (hunk 3 above)
Comment (namrata1012) [reply www/src/pages/table/NodeSearchComponent.tsx:74]: We are using the Css we add in that component

**Pull request merged**